README.md

```diff
@@ -1,32 +1,143 @@
+    ...
+            
+    //constructors
+    ...
+            
+    //methods
+    ...
+    
+}
+```
+
+Here are a few things to note:
+- The `public` keyword is covered in a later lesson on **access modifiers**.
+- The `class` keyword indicates we are defining a new Java class.
+- The class name should follow standard naming conventions, which are described in a subsequent section.
+- The open curly brace `{` indicates the beginning of the class definition and
+  has a matching close curly brace `}` at the end of the class definition.
+  The area between the curly braces is called the **class body**. The class body contains:
+  - fields (variables) to store the state of the class and its objects.
+  - constructors to initialize the state of new objects by assigning values to the fields.
+  - methods to implement the behavior of the class and its objects.
+
+In this lesson, we will focus on defining fields to encapsulate state.
+The syntax and implementation of constructors and methods will be covered in a later lesson.
+
+- A field is a variable that stores a property of an object or the class itself.
+  - An **instance variable** stores a value in each instance of the class (i.e. each object has its own copy of the field).
+  - A **static variable** (also called a **class variable**) stores a single value in the class (i.e. there is only one copy of the field).
+- A field is normally defined on its own line of code with a `;` at the end.
+- A field is defined using a variable declaration that has a "type" and a "name", such as `String breed;`.
+  - The type allows us to tell Java what type of value we expect this property to be (i.e. `int`, `String`, etc).
+  - The name is how we want to be able to refer to the property.
+- A field may have additional **access and non-access modifiers**, which are covered in later lessons.
+
+## Defining a `Dog` Class
+
+Consider how we might define a Java class to store data about two dogs: an 8-year-old Great Dane
+named Bruno, and a 5-year-old Pug named Penny.
+Both dogs have the same data structure (i.e. set of properties such as name, breed, etc.)
+but may have different values for each property ("Bruno" vs "Penny", "Great Dane" vs "Pug", etc).
+
+![dog objects](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/2dog_objects.png)
+
+Let's define a class named `Dog` with fields (instance variables) to store their name, breed, age, and whether their tail is wagging:
+
+```java
+public class Dog {
+    
+    String name;
+    String breed;
+    int age;
+    boolean waggingTail;
+    
+} 
+```
+
+Constructors and methods are omitted from the current class definition.
+Eventually we will add methods that operate on the fields, such as a method
+named `eatTreat()` that assigns the `waggingTail` field to true.
+
+Once we have defined the `Dog` class, we can create many instances of the class.
+The visualization above depicts how memory is allocated to store the state of two `Dog`
```

Comment:
It looks like in the outline we have the memory management and garbage collection lesson a little further down. Should we at least state (or is it assumed to have already been stated/explained?) that a computer stores these objects in something called "memory"?

Reply:
Will you be using the Java Visualizer in module 1 to show how variables are stored in the call stack with each method invocation?  That would probably be useful for helping folks understand local variables and parameter passing and how memory is managed for primitive values.  If not, I'll update this lesson to cover that.